server/app/services/CfJsonDocumentContext.java

```diff
@@ -218,6 +219,31 @@ public void putRepeatedEntities(Path path, ImmutableList<String> entityNames) {
     }
   }
 
+  /**
+   * Puts an array at a given path, building parent objects as needed.
+   *
+   * @param path the {@link Path} where the array should be added.
+   * @param entityNames a {@link List} containing service area results.
+   */
+  public void putServiceAreaInclusionEntities(
+      Path path, ImmutableList<ServiceAreaInclusion> entityNames) {
```

Comment:
Optional nit: It isn't clear to me why this is called entityNames since it seems to include a lot more.